qemu/tests/blockdev_commit_standby.py

```diff
@@ -0,0 +1,41 @@
+from provider import backup_utils
+from provider import job_utils
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitStandby(BlockDevCommitTest):
+    def commit_snapshots(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device)
+        self.main_vm.monitor.cmd(cmd, args)
+        job_id = args.get("job-id", device)
+        job_utils.wait_until_job_status_match(self.main_vm, "ready", job_id, timeout=120)
+        self.main_vm.monitor.cmd("job-pause", {"id": job_id})
+        if not job_utils.is_block_job_paused(self.main_vm, job_id):
+            self.test.fail("Block job is not paused")
+        self.main_vm.monitor.cmd("job-complete", {"id": job_id})
+        self.main_vm.monitor.cmd("job-resume", {"id": job_id})
+        if not job_utils.get_event_by_condition(self.main_vm, "BLOCK_JOB_COMPLETED"):
```

Comment:
It's better to use the job_id to filter the job, but here, it's OK for there is only one.